packages/kbn-rule-data-utils/src/technical_field_names.ts

```diff
@@ -17,25 +17,18 @@ const CONSUMERS = `${KIBANA_NAMESPACE}.consumers` as const;
 const ECS_VERSION = 'ecs.version' as const;
 const EVENT_ACTION = 'event.action' as const;
 const EVENT_KIND = 'event.kind' as const;
-const RULE_CATEGORY = 'rule.category' as const;
```

Comment:
`rule.*` should only refer to source/event document fields, not our kibana rules.